components/molecule/photoUploader/src/SkeletonCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const SKELETON_CLASS_NAME = `${BASE_CLASS_NAME}-skeleton`
+
+const SkeletonCard = ({icon, text}) => {
+  return (
+    <li className={SKELETON_CLASS_NAME}>
+      <div className={`${SKELETON_CLASS_NAME}Icon`}>
```

Comment:
`className={`${SKELETON_CLASS_NAME}-icon` instead of `className={`${SKELETON_CLASS_NAME}Icon`, no?